src/alias/azext_alias/__init__.py

```diff
@@ -0,0 +1,60 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import timeit
+
+from knack.log import get_logger
+
+from azure.cli.core import AzCommandsLoader
+from azure.cli.core.commands import CliCommandType
+from azure.cli.core.commands.events import EVENT_INVOKER_PRE_CMD_TBL_TRUNCATE
+from azext_alias.alias import AliasManager
+from azext_alias._const import DEBUG_MSG_WITH_TIMING
+from azext_alias.telemetry import telemetry
+
+logger = get_logger(__name__)
+VERSION = '0.1.0'
```

Comment:
I don't like having the same version no. defined both here and in setup.py.

Reply:
Consider doing something like what Python SDK does at https://github.com/Azure/azure-sdk-for-python/blob/master/azure-mgmt-cdn/setup.py#L45.

Alternatively, call `from azure.cli.core.extension import get_extension`. Then call it with the extension name then you can get your version that way.